readme.md

```diff
@@ -36,6 +36,14 @@ Any `.node` files included will also support binary relocation.
           options: {
             // optional, base folder for asset emission (eg assets/name.ext)
             outputAssetBase: 'assets',
+            // optional, restrict asset emissions to only the given folder.
+            filterAssetBase: process.cwd(),
+            // optional, permit entire __dirname emission
+            // eg `const nonAnalyzable = __dirname` can emit everything in the folder
+            emitDirnameAll: false,
+            // optional, permit entire filterAssetBase emission
+            // eg `const nonAnalyzable = process.cwd()` can emit everything in the cwd()
```

Comment:
I'm not sure I understand the `nonAnalyzable` part here. Where is that defined?

Reply:
This is just an example of how an asset reference can be passed in ways analysis can't trace, so that the statement above is enough to result in an emission.